src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -31,7 +31,7 @@ internal sealed class ManagedProjectSystemPackage : AsyncPackage
         protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
             IComponentModel componentModel = await this.GetServiceAsync<SComponentModel, IComponentModel>();
-
+            
```

Comment:
Doesn't seem like there's changes in this file, revert file to original.

Reply:
Done :)